models/user.go

```diff
@@ -915,6 +903,17 @@ func CreateUser(u *User) (err error) {
 		return err
 	}
 
+	// insert email address
+	if _, err := sess.Insert(&EmailAddress{
+		UID:         u.ID,
+		Email:       u.Email,
+		LowerEmail:  strings.ToLower(u.Email),
```

Comment:
u.Email = strings.ToLower(u.Email) at line 878

Reply:
It does appear that EmailAddress isn't doing this lowercasing so perhaps we need to do something about not lowercasing before we stick things into EmailAddress even if we have to lowercase in User.

Reply:
Done. Since user's email will be removed in next PRs, I think we can keep it there in this PR.